BurnOutSharp/ProtectionType/OpenMG.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using BurnOutSharp.ExecutableType.Microsoft.PE;
+using BurnOutSharp.Interfaces;
+using BurnOutSharp.Matching;
+
+namespace BurnOutSharp.ProtectionType
+{
+    /// <summary>
+    /// OpenMG is a form of DRM created by Sony to control how music is copied and listened to on PC. 
+    /// It is known to be used with multiple CD audio protections, such as XCP, LabelGate, and quite possibly Key2AudioXS.
+    /// References:
+    /// https://en.wikipedia.org/wiki/OpenMG
+    /// </summary>
+    public class OpenMG : IPathCheck, IPortableExecutableCheck
+    {
+        public string CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)
+        {
+            // Get the sections from the executable, if possible
+            var sections = pex?.SectionTable;
+            if (sections == null)
+                return null;
+
+            // Found in many different OpenMG related files ("Touch" by Amerie).
+            string name = pex.LegalTrademarks;
+            if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("OpenMG", StringComparison.OrdinalIgnoreCase))
+                return $"OpenMG";
+
+            // Found in "OMGDBP.OCX" ("Touch" by Amerie).
+            name = pex.FileDescription;
```

Comment:
If you're using the `FileDescription` field multiple times in a row, you don't have to keep assigning to `name`.